grunttasks/jshint.js

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  'use strict';
+
+  grunt.config('jshint', {
+    files: [
+      "{,bin/,config/,db/,scripts/,test/**/}*.{js,json}"
```

Comment:
Similar to the [previous comment](https://github.com/mozilla/fxa-auth-db-server/pull/132/files#r27606599), I think this path is a copy-pasta from somewhere else (my bad).

Me thinks it should just be:
```js
files: [
  "{,scripts/,test/**/}*.{js.json}"
]
```

Plus, not sure if we want to normalize the single-vs-double quotes. I thought JSHint would have caught that, but maybe we don't check in this repo or JSHint stopped supporting that config and pushed people towards JSCS... Again, probably not worth the herculean effort for these Grunt config files.